ref/Lists_of_Geometric_Elements.md

```diff
@@ -18,8 +18,10 @@ This is done with operators that return lists of elements of a particular kind.
 
 #### All springs of the construction: `allsprings()`
 
-#### All elements incident to a given one: `allsegments(‹geo›)`
+#### All elements incident to a given one: `all*(‹geo›)`
```

Comment:
Can you please spell these out explicitely? We may want to use the reference manual to generate a list of all supported functions, to be used in e.g. an upcoming CindyJS editor. For that we need the list to be machine-parseable.